ui/src/layout.tsx

```diff
@@ -16,6 +15,8 @@ const iconStyles = {
 };
 
 const Header = () => {
+    const { projectId = '' } = useParams<{ projectId: string }>();
```

Comment:
Perhaps extract these into a `useProjectIdentifier()` hook like we do in Geti.
We could add an extra guard/assert there to make sure `projectId` is not empty.

Reply:
Updated